src/main/resources/templates/macros/report/docstring.vm

```diff
@@ -3,10 +3,11 @@
 #if ($docString)
   <div class="inner-level">
     <div class="docstring indention">
-      <div data-toggle="collapse" class="collapsable-control" data-target="#msg-$docString.hashCode()">
+      #set($docstringId = $counter.getNext())
+      <div data-toggle="collapse" class="collapsable-control" data-target="#msg-$docstringId">
```

Comment:
if nextValue() returns string with prefix "msg-" than this prefix does not have to be added to each occurrence